src/index.html

```diff
@@ -8,6 +8,78 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+    <div>
+      <div class="stars stars--0">
+        <div class="stars__star stars__star-inactive"></div>
```

Comment:
<img width="844" alt="Screenshot 2023-08-02 at 16 12 23" src="https://github.com/mate-academy/layout_stars/assets/53228259/1f49f86b-1a50-4460-bb3d-429036f90a32">
```suggestion
        <div class="stars__star"></div>
```

Reply:
Adjusted the previous element of “stars__star-inactive” to “stars__star--inactive” to match the modifier styling.

Reply:
Add one of the next extra classes (a modifier) stars--0, stars--1, stars--2 ... stars--5 to each block

you already added this modifier line before

Reply:
I just tried adding the stars--0 modifier to each of the blocks, in addition to the addition to the modifier in line 12, and got numerous errors. 
<img width="1800" alt="image" src="https://github.com/mate-academy/layout_stars/assets/32708111/4832a780-01a0-4e97-881d-aebb69f58046">

Reply:
Reclassified the parent div as "stars" and then the subsequenqent child blocks were given the modifiers of "stars--0" (...).
Also restructured the CSS element to attribute "display: flex" and "flex-direction: row" to each of the stars--N modifiers

Reply:
if this still misses the requirements, would you possibly have a different example of what this question is looking for as I am not 100% sure what I should be doing to fully fill the requirements of the task.